flink-java/src/main/java/org/apache/flink/api/java/io/PrintingOutputFormat.java

```diff
@@ -18,12 +18,16 @@
 
 package org.apache.flink.api.java.io;
 
-import java.io.PrintStream;
-
 import org.apache.flink.annotation.PublicEvolving;
 import org.apache.flink.api.common.io.RichOutputFormat;
 import org.apache.flink.configuration.Configuration;
 
+import java.io.PrintStream;
+
+/**
+ * Output format that prints results into underlying {@link PrintStream}.
```

Comment:
I would reword this to mention stdout/stderr; the usage of `PrintStream` is an (unimportant) implementation detail.